setup/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles.csproj

```diff
@@ -1,24 +1,32 @@
 ﻿<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
 <Project Sdk="Microsoft.NET.Sdk">
+
   <Import Project="..\..\build\import\VisualStudio.props" />
-  
+
   <PropertyGroup>
+    <!-- VSIX -->
+    <VsixSigningDependencyTarget>SwixBuild</VsixSigningDependencyTarget>
```

Comment:
What's this?

Reply:
In my description in the PR, I describe this in the `MSBuild Properties` section.

Reply:
A more idomatic name would be `AddVsixForSigningDependsOn`. See the MSBuild common targets for other examples.

Reply:
This has been renamed to `AddVsixForSigningDependsOn`. Thanks!